### ISSUE TITLE
[feat] Allow tracking of a standalone project w/o workspace

### ISSUE THREAD

**Issue by vhespanha** [GnikDroy/projections.nvim#45]
Hey there,

I've got a bit of a unique use case. I don't want Projections to track all git repos in my home folder, but I do need it to track one specific repo (my dotfiles repo located at `~/dotfiles`). 

I couldn't find a way to specify a standalone project like that—only to specify workspaces. I know there are workarounds, but I think implementing this feature would give the plugin more flexibility.

It would be great to have syntax to expose only one project from a directory without a parent workspace tracking it.

I'll probably submit a PR this weekend to take a swing at implementing this.

**Comment by kaixuan188** [comment]
Yes, this is exactly what I need. I think there should be a separate option to set a directory as a project... Is it really not available after all this time?

**Comment by GnikDroy** [comment]
Sorry haven't been able to look at this. I can accept PRs.
